planner/core/find_best_task.go

```diff
@@ -289,6 +289,115 @@ func (p *baseLogicalPlan) enumeratePhysicalPlans4Task(physicalPlans []PhysicalPl
 	return bestTask, cntPlan, nil
 }
 
+func (p *LogicalSequence) enumeratePhysicalPlans4Task(physicalPlans []PhysicalPlan,
```

Comment:
`(p *LogicalSequence) enumeratePhysicalPlans4Task` and `(p *baseLogicalPlan) enumeratePhysicalPlans4Task` is 80% the same, I think it's better not to copy it.

Reply:
But is there a good way to merge them? I haven't come up with one. So the codes are like current.

Reply:
You can just check if `p.self` is a `LogicalSequence` in `(p *baseLogicalPlan) enumeratePhysicalPlans4Task`.